src/accountlist/accountlistReducers.js

```diff
@@ -0,0 +1,57 @@
+import { fromJS } from 'immutable';
+
+import {
+  REALM_ADD,
+  LOGIN_SUCCESS,
+  LOGOUT,
+  ACCOUNT_REMOVE,
+} from '../account/accountActions';
+
+const initialState = fromJS([]);
+
+export default (state = initialState, action) => {
```

Comment:
Oh ok. How about adding a timestamp then and sorting in the UI component? I'm not a big fan of coupling UI (such as 'sorted by most recent') with the data model.

I think the logic for `REALM_ADD` and `LOGIN_SUCCESS` below is too complex.